vowpalwabbit/cb_explore_adf_common.h

```diff
@@ -123,16 +123,16 @@ inline void cb_explore_adf_base<ExploreType>::predict(
   if (label_example != nullptr)
   {
     // predict path, replace the label example with an empty one
-    data._action_label = label_example->l.cb;
-    label_example->l.cb = data._empty_label;
+    data._action_label = std::move(label_example->l.cb);
+    label_example->l.cb = std::move(data._empty_label);
   }
 
   data.explore.predict(base, examples);
 
   if (label_example != nullptr)
   {
     // predict path, restore label
-    label_example->l.cb = data._action_label;
+    label_example->l.cb = std::move(data._action_label);
```

Comment:
Shouldn't `data._empty_label` be moved back into down here?

Reply:
Just changed this in the last commit

Reply:
@jackgerrits should we also reset the label here? i.e. clear the array and reset the cost?

Reply:
Yeah we probably should